### PR TITLE
feat: tabs added into component library

### PR DESCRIPTION
**Pull request by sonishreyas**
## Changes logs
Tabs added into component library

## Screenshots
image
image

## Preview Link
https://deploy-preview-11--ripple-ui.netlify.app/components/tabs/tabs.html

### CODE REVIEW

**Comment by rishimish** [comment]
The tabs look good and functional.
Just one thing - In the vertical tab, size changes when clicked three.

**Review by rishmi5h — COMMENTED** [review]
The tabs look good and functional.
Just one thing - In the vertical tab, size changes when clicked three.

**Comment by sonishreyas** [comment]
> The tabs look good and functional. Just one thing - In the vertical tab, size changes when clicked three.

Thanks, got the fix will push the changes

File: components/tabs/src/css/tabs.css [hunk 1]
```diff
@@ -59,26 +59,57 @@ body {
...
 	border: 0;
 	border-right: 0.05rem solid var(--primary-color);
+	margin: 1rem 0;
 }
 
 .vertical-tabs-btn-active {
 	padding-right: 3rem;
 	font-weight: bold;
-	border-right: 0.25rem solid var(--cta-color);
+	border-right: 0.5rem solid var(--cta-color);
```
Comment (sach10-create) [inline components/tabs/src/css/tabs.css:78]: less than 1 rem, can be changed to px

**Review by sach10-create — APPROVED** [review]
Good to merge.

**Pull request merged**